static/css/game-events.css

```diff
@@ -1,37 +1,69 @@
-.filters-container {
-    margin-bottom: 20px;
-    padding: 20px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    width: 20%;
+.form-group1 {
+    display: grid;
+    grid-template-columns: repeat(5 , 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 50px;
+    grid-row-gap: 0px;
 }
 
-.form-group {
-    margin-bottom: 10px;
-}
+.input-group1 { grid-area: 2 / 1 / 3 / 2; }
+.input-group2 { grid-area: 2 / 2 / 3 / 3; }
+.input-group3  { grid-area: 2 / 3 / 3 / 4; }
+.button1 { grid-area: 2 / 4 / 3 / 5; }
+.button2 { grid-area: 2 / 5 / 3 / 6; }
```

Comment:
Can we make those names more informative? Imagine you need to change them, it will be easier to know what you touch if the names have meaning.

Reply:
What about the input-groups? Can we give them meaningful names as well?